install-oracle.sh

```diff
@@ -254,6 +254,13 @@ SWAP_BLK_DEVICE_PARAM=".*"
 COMPATIBLE_RDBMS="${COMPATIBLE_RDBMS:-0}"
 COMPATIBLE_RDBMS_PARAM="^[0-9][0-9]\.[0-9].*"
 
+INSTALL_WORKLOAD_AGENT="${INSTALL_WORKLOAD_AGENT:-FALSE}"
```

Comment:
Rather than accepting a string value, defaulting, and validating, it might be cleaner to handle as a boolean, as we do for --no-patch.  It also allows a more fluent command-line.